src/utills.js

```diff
@@ -0,0 +1,47 @@
+import { RESULT_TEXT } from "./constants.js";
+
+export const generateRandomNumber = () => {
+  return MissionUtils.Random.pickNumberInRange(0, 9);
+};
+
+export const printResultOneTurn = (cars) => {
+  const app = document.getElementById("app");
+  const resultAreaP = document.createElement("p");
+  let resultCarText = "";
+
+  for (let i = 0; i < cars.length; i += 1) {
+    resultCarText += `${cars[i].carName}: ${"-".repeat(cars[i].forwardCount)}\n`;
+  }
+
+  resultAreaP.innerText = resultCarText;
+  app.appendChild(resultAreaP);
+};
+
+export const getWinner = (cars) => {
+  let winnerList = [];
+  let largestForwardCount = 0;
+
+  for (let i = 0; i < cars.length; i += 1) {
+    if (cars[i].forwardCount > largestForwardCount) {
+      largestForwardCount = cars[i].forwardCount;
+      winnerList = [cars[i].carName];
+    } else if (cars[i].forwardCount === largestForwardCount) {
+      winnerList.push(cars[i].carName);
+    }
+  }
+
+  return winnerList;
+};
```

Comment:
`largestForwardCount` 를 찾는 로직과 
`car.forwardCount === largestForwardCount` 를 찾는 로직이 섞여 있네요

프론트엔드에서는 일반적으로 엄청나게 큰 숫자를 대상으로 한 계산을 하지 않기 때문에, 
성능 최적화보다는 가독성에 더 초점을 맞춰서 코드를 작성하시는게 좋습니다 :smile: 


```suggestion
export const getWinner = (cars) => {
  const largestForwardCount = Math.max(...cars.map(({forwardCount}) => forwardCount));
  // const largestForwardCount = cars.reduce((max, {forwardCount}) => Math.max(max, forwardCount), 0);
  const winnerList = cars.filter(({forwardCount}) => forwardCount === largestForwardCount);
  
  return winnerList;
};
```

Reply:
아하 이렇게 활용할 수도 있군요..! 수정하겠습니다.